lib/config/index.js

```diff
@@ -24,7 +24,7 @@
 const assert = require('assert');
 
 const rc = require('rc');
-
+const KuzzleConfiguration = require('../config/defaultTsConfig');
```

Comment:
You should rather import what you need using the destructuring assignment: `const { defaultConfig } = require(...)`.  
It allows to keep consistent names across the codebase